scripts/generator.py

```diff
@@ -1,234 +1,234 @@
-        site = objectify.Element("site", nsmap='', _pytype='')
-        site.domain = item["domain"]
-        site.notes = item["notes"]
-        site.path = item["path"]
-        site.reason = item["reason"]
-
-        sites.append(site)
-        
-    objectify.deannotate(sites)
-    etree.cleanup_namespaces(sites)
-    return str(etree.tostring(sites, pretty_print=True, xml_declaration=True, with_tail=False), "utf-8")
-
-print("..... Generating master (sites.xml) list")
-with open("dist/sites.yaml", "r") as f1:
-    with open("dist/sites.xml", "w", encoding="utf-8") as f2:
-        f2.write(convertToXML(f1.read()))
-
-for item in all_lists:
-    game = item.replace(".yaml", "")
-    print(f"..... Generating {game}.xml list")
-    with open("lists/" + item, "r") as f1:
-        with open("dist/" + game + "/sites.xml", "w", encoding="utf-8") as f2:
-            f2.write(convertToXML(f1.read()))
-
-
-# ---------------------------------------------------- shields
-# Generates Shields which display stats
-print("[7/8] Generating SHIELDS")
-
-print("..... Generating total shield")
-with open("dist/sites.yaml", "r") as f1:
-    with open("dist/stats/total.json", "w", encoding="utf-8") as f2:
-        data = next(yaml.load_all(f1.read(), Loader=yaml.FullLoader))
-        sites = len(data)
-        total = {"schemaVersion": 1,
-                 "label": "sites",
-                 "message": str(sites),
-                 "color": "blue"}
-        f2.write(json.dumps(total))
-        
-print("..... Generating refreshed shield")
-with open("dist/stats/refreshed.json", "w", encoding="utf-8") as f:
-    refreshed = {"schemaVersion": 1,
-                 "label": "refreshed",
-                 "message": str(current_time) + " UTC",
-                 "color": "blue"}
-    f.write(json.dumps(refreshed))
-        
-# ---------------------------------------------------- index
-# Generates the main HTML frontend of the API
-print("[8/8] Generating INDEX")
-
-print("..... Generating index.html")
-with open("templates/index.html", "r") as f1:
-    with open("dist/index.html", "w", encoding="utf-8") as f2:
-        f2.write(f1.read().replace("{0}", str(current_time)))
-        
-print("..... Generating _redirects")
-with open("dist/_redirects", "w", encoding="utf-8") as f:
-    # More info about the _redirects file: https://developers.cloudflare.com/pages/platform/redirects
-    f.write("""/docs https://github.com/StopModReposts/Illegal-Mod-Sites/wiki/API-access-and-formats""")
-    
-print("..... Generating _headers")
-with open("dist/_headers", "w", encoding="utf-8") as f:
-    # More info about the _headers file: https://developers.cloudflare.com/pages/platform/headers
-    f.write("""/*\n  Access-Control-Allow-Origin: *""")
-    
-end_time = time.time()
-elapsed_time = str(end_time - start_time)
-
-print("[ ✓ ] Generated all formats")
-print(f"..... took {elapsed_time} seconds")
\ No newline at end of file
+print("[...] Initializing static generator")
```

Comment:
As for this rename to scripts folder, I think it's cleaner, but would require reconfiguration of cloudflare pages before merge.